### ISSUE TITLE
NTR: Liver biopsy: uridine diphosphate glucuronosyltransferase activity reduced (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9091]
New term request Liver biopsy: uridine diphosphate glucuronosyltransferase activity reduced (segal_060324181441): Hepatic biopsy

**Comment by pnrobinson** [comment]
@MickeySegal
It seems this is not liver specific?
https://omim.org/entry/230350

**Comment by MickeySegal** [comment]
Actually https://omim.org/entry/606785 and measured in liver but not clear if only liver.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000617
- label: Reduced tissue UDP-glucuronyl-transferase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:9869613
- def: Concentration or activity of uridine diphosphate glucuronosyltransferase below the lower limit of normal. The activity of this enzyme can be measured in liver and rarely in other tissues.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9091
- synonyms: Reduced tissue uridine diphosphate glucuronosyltransferase activity(NONE)|Reduced liver uridine diphosphate glucuronosyltransferase activity(NONE)
